test/call-test.js

```diff
@@ -807,8 +805,9 @@
                 assert.equals(object.doIt.getCall(0).toString().replace(/ at.*/g, ""), "doIt() !TypeError(Oh noes!)");
             },
 
+            // these tests are ensuring that call.toString is tightly coupled to sinon.format
```

Comment:
Do you see a way we could we rewrite the tests somehow to avoid the tight string formatting coupling?